### ISSUE TITLE
setup.py fails due to non existing wheezy backports RELEASE file

### ISSUE THREAD

**Issue by jvijtiuk** [panda-re/lava#27]
Hello,

while building LAVA on Ubuntu 16.04 the following error appears:

```
[setup.py] Running [docker build -t lava32 /home/user/lava2/docker] . . . 
Sending build context to Docker daemon  10.24kB
Step 1/53 : FROM i386/debian:stretch
 ---> 780c163f5cc0
Step 2/53 : RUN echo deb http://httpredir.debian.org/debian wheezy-backports main >> /etc/apt/sources.list
 ---> Using cache
 ---> 0376a9afd0e3
Step 3/53 : RUN apt-get update
 ---> Running in e7e7f3f83bcd
Get:1 http://security-cdn.debian.org/debian-security stretch/updates InRelease [94.3 kB]
Ign:2 http://cdn-fastly.deb.debian.org/debian wheezy-backports InRelease
Ign:3 http://cdn-fastly.deb.debian.org/debian stretch InRelease
Get:4 http://cdn-fastly.deb.debian.org/debian stretch-updates InRelease [91.0 kB]
Ign:5 http://cdn-fastly.deb.debian.org/debian wheezy-backports Release
Get:6 http://cdn-fastly.deb.debian.org/debian stretch Release [118 kB]
Ign:7 http://httpredir.debian.org/debian wheezy-backports/main i386 Packages
Ign:7 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main i386 Packages
Ign:8 http://httpredir.debian.org/debian wheezy-backports/main all Packages
Ign:8 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main all Packages
Get:9 http://cdn-fastly.deb.debian.org/debian stretch Release.gpg [2434 B]
Ign:7 http://httpredir.debian.org/debian wheezy-backports/main i386 Packages
Ign:7 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main i386 Packages
Ign:8 http://httpredir.debian.org/debian wheezy-backports/main all Packages
Ign:8 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main all Packages
Get:10 http://security-cdn.debian.org/debian-security stretch/updates/main i386 Packages [487 kB]
Ign:7 http://httpredir.debian.org/debian wheezy-backports/main i386 Packages
Ign:7 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main i386 Packages
Ign:8 http://httpredir.debian.org/debian wheezy-backports/main all Packages
Ign:8 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main all Packages
Ign:7 http://httpredir.debian.org/debian wheezy-backports/main i386 Packages
Ign:7 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main i386 Packages
Ign:8 http://httpredir.debian.org/debian wheezy-backports/main all Packages
Ign:8 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main all Packages
Get:11 http://cdn-fastly.deb.debian.org/debian stretch-updates/main i386 Packages [11.4 kB]
Ign:7 http://httpredir.debian.org/debian wheezy-backports/main i386 Packages
Ign:7 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main i386 Packages
Ign:8 http://httpredir.debian.org/debian wheezy-backports/main all Packages
Ign:8 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main all Packages
Get:12 http://cdn-fastly.deb.debian.org/debian stretch/main i386 Packages [7062 kB]
Err:7 http://httpredir.debian.org/debian wheezy-backports/main i386 Packages
  404  Not Found
Ign:7 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main i386 Packages
Ign:8 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main all Packages
Fetched 7866 kB in 7s (995 kB/s)
Reading package lists...
W: The repository 'http://httpredir.debian.org/debian wheezy-backports Release' does not have a Release file.
E: Failed to fetch http://httpredir.debian.org/debian/dists/wheezy-backports/main/binary-i386/Packages  404  Not Found
E: Some index files failed to download. They have been ignored, or old ones used instead.
The command '/bin/sh -c apt-get update' returned a non-zero code: 100

[setup.py] [docker build -t lava32 /home/user/lava2/docker] cmd did not execute properly.
```

The same issue appeared when building LAVA on Debian 9.8.

I managed to successfully build LAVA by removing the following line from docker/Dockerfile:
```
RUN echo deb http://httpredir.debian.org/debian wheezy/backports main >> /etc/apt/sources.list
```

However, I'm not sure if that is an appropriate solution, especially since no similar issues have been reported so far.

**Comment by moyix** [comment]
It looks like this is because wheezy-backports was recently removed from the main debian mirrors – could you try the change suggested here and see if it works?

https://unix.stackexchange.com/a/508728/95861

**Comment by jvijtiuk** [comment]
Changing the mirror to archive.debian.org seems to work. I've opened a pull request with the fix (#28) and am closing the issue.
